java/src/main/java/svix/SvixOptions.java

```diff
@@ -0,0 +1,19 @@
+package svix;
+
+public class SvixOptions {
+	private final boolean debug;
+	private final String testUrl;
```

Comment:
can we rename this to `debugUrl` to match our go library

Reply:
Done. Note that this is the name in the python implementation.

Reply:
Done. Note that this is the name in the python implementation.